nextflow.config

```
@@ -311,8 +316,12 @@ profiles {
     variantcalling_channels    { includeConfig 'conf/test/variantcalling_channels.config' }
 }
 
-// Load nf-core custom profiles from different Institutions
-includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/nfcore_custom.config" : "/dev/null"
+// Load nf-core custom profiles from different institutions
+
+// If params.custom_config_base is set AND either the NXF_OFFLINE environment variable is not set or params.custom_config_base is a local path, the nfcore_custom.config file from the specified base path is included.
+// Load nf-core/sarek custom profiles from different institutions.
+includeConfig params.custom_config_base && (!System.getenv('NXF_OFFLINE') || !params.custom_config_base.startsWith('http')) ? "${params.custom_config_base}/nfcore_custom.config" : "/dev/null"
+
 
 // Load nf-core/sarek custom profiles from different institutions.
 includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/pipeline/sarek.config" : "/dev/null"
```

This was also changed in the template
```suggestion
includeConfig params.custom_config_base && (!System.getenv('NXF_OFFLINE') || !params.custom_config_base.startsWith('http')) ? "${params.custom_config_base}/pipeline/sarek.config" : "/dev/null"
```